packages/components/icons/src/templates/icon.styles.tsx

```diff
@@ -15,14 +15,18 @@ export type Props = {
     | 'warning'
     | 'error'
     | 'success';
-  size?: 'small' | 'medium' | 'big' | 'scale';
+  size?: 'small' | 'medium' | 'big' | 'scale' | '10' | '20' | '30' | '40';
```

Comment:
Perhaps we can add JSDocs here and explain the t-shirt sizes are deprecated.

Reply:
addressed [here](https://github.com/commercetools/ui-kit/pull/2830/files#diff-e2b96990d679733055c76ed55fb98bd14d8d66deddd6f44eb8511c64d7ec9636R19)